assignments/arrays.js

```diff
@@ -63,32 +63,65 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let forgottenCar = inventory[32];
+console.log("Car 33 is a " + forgottenCar.car_year + " " + forgottenCar.car_make + " " + forgottenCar.car_model + ".");
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[49];
+console.log(lastCar.car_make + " " + lastCar.car_model);
+
+//{"id":50,"car_make":"Lincoln","car_model":"Town Car","car_year":1999}];
```

Comment:
This definitely works but using `inventory[49]` is too hardcoded so if a car was added to the list, 49 would no longer be the last one. A better way that works no matter how many cars are in the array would be `let lastCar = inventory[inventory.length-1];`